deploy/openj9-chart/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+OpenJ9 Helm Chart
```

Comment:
Can this file be deleted?

Reply:
This entire file is printed to the terminal in the `helm` command as shown in the last line. 
```
$ helm install my-release openj9-chart/
NAME: my-release
LAST DEPLOYED: Tue Sep 15 14:51:45 2020
NAMESPACE: helm-chart-test
STATUS: deployed
REVISION: 1
NOTES:
OpenJ9 Helm Chart
```
Instead of deleting it, I will add more information about the helm chart to this file so that people can see after installing the chart.